### PR TITLE
Add primary terminal hook

### PR DESCRIPTION
**Pull request by tomquirk**
Add a hook for fetching the projects primary ETH payment terminal.

Some other minor refactoring in the PR too coz i'm too lazy to split it up.

Closes https://github.com/jbx-protocol/juice-interface/issues/1382

### CODE REVIEW

File: src/components/v2v3/V2V3Project/V2V3Project.tsx [hunk 1]
```diff
@@ -158,25 +141,11 @@ export function V2V3Project() {
...
 
   return (
     <Space direction="vertical" size={GUTTER_PX} style={{ width: '100%' }}>
-      {showV2BugNoticeBanner || showRelaunchFundingCycleBanner ? (
-        <Space direction="vertical" size="small" style={{ width: '100%' }}>
-          {showV2BugNoticeBanner && <V2BugNoticeBanner />}
-          {showRelaunchFundingCycleBanner && <RelaunchFundingCycleBanner />}
-        </Space>
-      ) : null}
+      <ProjectBanners />
```
Comment (tomquirk) [inline src/components/v2v3/V2V3Project/V2V3Project.tsx:148]: Random unrelated cleanup

File: src/components/v2v3/V2V3Project/banners/ProjectBanners.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
+import { Space } from 'antd'
```
Comment (tomquirk) [inline src/components/v2v3/V2V3Project/banners/ProjectBanners.tsx:1]: random unrelated cleanup

File: src/components/v2v3/V2V3Project/V2V3Project.tsx [hunk 3]
```diff
@@ -34,14 +30,23 @@ import { V2V3ProjectHeaderActions } from './V2V3ProjectHeaderActions/V2V3Project
...
+    useState<boolean>(false)
+
   return (
-    <TextButton
-      onClick={onClick}
-      style={{ fontWeight: 400, fontSize: '0.8rem' }}
-    >
-      <Trans>All assets</Trans>
-    </TextButton>
+    <>
```
Comment (tomquirk) [inline src/components/v2v3/V2V3Project/V2V3Project.tsx:38]: Random unrelated cleanup

File: src/components/Project/PayProjectForm/payProjectFormContext.tsx [hunk 4]
```diff
@@ -1,9 +1,13 @@
 import { BigNumber } from '@ethersproject/bignumber'
-import { createContext } from 'react'
+import { createContext, CSSProperties } from 'react'
 import { WeightFunction } from 'utils/math'
-import { PayButtonProps } from './PayProjectForm'
 import { PayProjectForm } from './usePayProjectForm'
 
+export interface PayButtonProps {
```
Comment (tomquirk) [inline src/components/Project/PayProjectForm/payProjectFormContext.tsx:6]: Moved dis here

File: src/hooks/v2v3/V2V3ProjectState.ts [hunk 5]
```diff
@@ -117,15 +118,17 @@ export function useV2V3ProjectState({ projectId }: { projectId: number }) {
   const { data: terminals } = useProjectTerminals({
     projectId,
   })
-  const primaryTerminal = terminals?.[0] // TODO: make primaryTerminalOf hook and use it
+  const { data: primaryETHTerminal } = useProjectPrimaryEthTerminal({
```
Comment (tomquirk) [inline src/hooks/v2v3/V2V3ProjectState.ts:121]: This is the main change

**Comment by tomquirk** [comment]
@wraeth-eth wdyt my fine ser?

**Review by wraeth-eth — APPROVED** [review]
